packages/notifications/src/inAppMessaging/providers/pinpoint/apis/setConflictHandler.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { InAppMessage } from '../../../types';
+import { InAppMessageConflictHandler, SetConflictHandlerInput } from '../types';
+
+/**
+ * Set a conflict handler that will be used to resolve conflicts that emerge
+ * when matching the event to the synced messages.
+ * This setting is not persisted across restarts and hence needs to be called before dipatching an event.
```

Comment:
Might make more sense as a remark

```suggestion
 * @remark
 * The conflict handler is not persisted between sessions and needs to be called before dispatching an event to have any effect.
```